specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-05-01-preview/migrate.json

```diff
@@ -17563,11 +17808,62 @@
         "MonthlyAzureHybridCostSavings",
         "MonthlySecurityCost",
         "MonthlyPremiumV2StorageCost",
-        "MonthlyLinuxAzureHybridCostSavings"
+        "MonthlyLinuxAzureHybridCostSavings",
+        "MonthlyAvsNodeCost",
```

Comment:
This seems to be used in multiple locations like AssedMachineSqlProperties and AssessedMachineProperties apart from AvsAssessmentProperties. Do we want to keep "Avs" in the name

Reply:
AvsNodes are itself a separate category of costs which will be used for avsAssessments.